hasher-matcher-actioner/hmalib/lambdas/fetcher.py

```diff
@@ -128,15 +115,17 @@ def lambda_handler(event, context):
             )
             continue
 
-        indicator_store = ThreatUpdateS3PDQStore(
+        indicator_store = ThreatUpdateS3Store(
             int(collab.privacy_group_id),
             api.app_id,
             te_data_bucket,
             config.s3_te_data_folder,
             config.data_store_table,
+            supported_signal_types=[VideoMD5Signal, PdqSignal],
```

Comment:
~nit: I don't understand having this be a param instead of hardcoding this here if we are also going to have:
```      
         indicator_types = [
            "HASH_VIDEO_MD5"
        ]  # hardcoding this because md5 currently points to HASH_MD5
```
in the other files anyway. Either made that block conditional on this or just have them be class values.